### PR TITLE
Add maxHealth ability (#237)

### LINKED ISSUE #237: Add ability to get warrior's max health
The proposal is to add a `.maxHealth()` ability (sense), that allows the player to get the warrior's max health programmatically, so hardcoding its value in the code is no longer necessary.

### PR DESCRIPTION
**Pull request by JSeed**
Added to towers baby-steps and tick-tick-boom.

Closes #237.

### CODE REVIEW

**Comment by JSeed** [comment]
Added maxHealth ability and included it in existing towers, updating the baby-steps tip to introduce it.

Let me know if I missed anything!

File: packages/warriorjs-tower-baby-steps/src/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {
   attack,
   feel,
   health,
+  maxHealth,
```
Comment (olistic) [inline packages/warriorjs-tower-baby-steps/src/index.js]: nit: this should be in alphabetical order.

**Comment by JSeed** [comment]
Thanks for the feedback!  The code base was a pleasure to work with.

Good catch, I'll push a fix shortly.

**Comment by olistic** [comment]
Thanks, and welcome to the Contributors!

**Pull request merged**